tests/unit/-private/table-sticky-polyfill-test.js

```diff
@@ -100,6 +109,36 @@ function verifyFooter(assert) {
     });
 }
 
+/**
+ * Verifies multi line header when scrolled to the bottom of the table
+ * @param assert
+ */
+function verifyMultiLineHeader(assert) {
+  let firstCellRect = find('thead tr:first-child th:first-child').getBoundingClientRect();
+  let expectedOffset = firstCellRect.top;
+
+  findAll('thead > tr').forEach(row => {
+    let firstCellRect = row.firstElementChild.getBoundingClientRect();
```

Comment:
Is `firstElementChild` a stable part of the DOM? Is there any better identifier (tag name, class name) that could be used to ensure that `firstCellRect` is what we think it is?

Reply:
Good call out that `firstElementChild` is not supported across all modern browsers as it is unsupported in IE8, IE9, and Safari. I've modified the search string of `findAll` from `thead > tr` to `thead > tr > td:first-child` so we are directly searching for the first table cell of each header row and made the same modifications to the footer verification function as well.